src/vtt-segment-loader.js

```diff
@@ -279,6 +279,16 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     this.mediaSecondsLoaded += segment.duration;
 
+  /**
+   * The below code helps in removing double captions
+   */
+    if (segmentInfo.cues.length) {
+      let lastCueEndTime = segmentInfo.cues[segmentInfo.cues.length - 1].endTime;
+      let firstCueEndTime = segmentInfo.cues[0].endTime;
+
+      this.remove(firstCueEndTime, lastCueEndTime);
```

Comment:
This is more of a preference thing, but you can remove the variables and just do it all in one line:

```
this.remove(segmentInfo.cues[0].endTime, segmentInfo.cues[segmentInfo.cues.length - 1].endTime)
```